src/modal_dialog/index.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component, PropTypes } from 'react';
+import _ from 'lodash';
+
+// React Utils
+import classNames from 'classnames';
+
+// Main Container Class
+export default class Modal extends Component {
+
+  static propTypes = {
+    id:            PropTypes.string,
+    closeable:     PropTypes.bool,
+    size:          PropTypes.oneOf(['small', 'medium', 'large']),
+    onClose:       PropTypes.func,
+    isModalOpen:   PropTypes.bool,
+    scrollingBody: PropTypes.bool,
+    lightbox:      PropTypes.bool,
+    keyboard:      PropTypes.bool
+  };
+
+  static defaultProps = {
+    id:            '',
+    closeable:     true,
+    size:          'medium',
+    onClose:       function() {},
+    isModalOpen:   true,
+    scrollingBody: false,
+    lightbox:      true,
+    keyboard:      true
+  };
+
+  constructor(props) {
+    super(props);
+
+    this._closeDialog = this._closeDialog.bind(this);
```

Comment:
yeah you have to do this if you're not using static property initializers, but we are so you can do instead

```javascript
_closeDialog = () => { 
  //...
}
```